Shared/Samples/Sketch on map/SketchOnMapView.swift

```diff
@@ -183,7 +183,8 @@ extension GeometryEditorMenu {
 }
 
 /// An object that acts as a view model for the geometry editor menu.
-@MainActor class GeometryEditorMenuModel: ObservableObject {
+@MainActor
+class GeometryEditorMenuModel: ObservableObject {
```

Comment:
Were you getting warnings on these? I'm not if I undo the changes.

Reply:
Are you using SwiftLint 0.51.0?

Reply:
I wasn't before but I updated and I still am not. It's okay though, no need to hold the PR for this.

Reply:
Scratch that, I just built again and the warnings finally came up 😌

Reply:
<img width="1021" alt="image" src="https://user-images.githubusercontent.com/9660181/233695054-a9fa4d59-a42a-4856-aa4c-7f8b41dd2ef1.png">

Hmm interesting. I don't know why these warnings popped up after I upgraded. The rule seems to be pretty old and I cannot find a related PR (might be https://github.com/realm/SwiftLint/pull/4616 ?)

I'll merge this PR as it is not giving warnings on your end.

Reply:
Hehe thanks for the update. 😬